### PR TITLE
updating phoenix_html

### PR DESCRIPTION
**Pull request by BlueHotDog**
upgrading libs

### CODE REVIEW

File: lib/kaffy/resource_form.ex [hunk 1]
```diff
@@ -73,10 +73,10 @@ defmodule Kaffy.ResourceForm do
     end
   end
 
-  def form_field(changeset, form, {field, options}, opts) do
-    type = Kaffy.ResourceSchema.field_type(changeset.data.__struct__, field)
-    build_html_input(changeset.data, form, {field, options}, type, opts)
-  end
+  # def form_field(changeset, form, {field, options}, opts) do
```
Comment (BlueHotDog) [inline lib/kaffy/resource_form.ex:76]: just removing non used function. didn't want to delete.

**Comment by colindensem** [comment]
Thanks @BlueHotDog would resolve #217 to some degree :)

**Comment by willjleong** [comment]
Thanks for this! Can we get this in a new version of `kaffy`? I have to use an older version of `phoenix_live_dashboard` because this hasn't been published yet.

**Comment by Tiuipuv** [comment]
@kelcecil any chance you can publish v0.9.1, since it includes this change, to https://hex.pm/packages/kaffy?

**Pull request merged**